### PR TITLE
Added test for image_recognition_tensorflow

### PR DESCRIPTION
**Pull request by reinzor**
Should close https://github.com/tue-robotics/image_recognition/issues/76

### CODE REVIEW

File: image_recognition_tensorflow/test/models/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*
```
Comment (MatthijsBurgh) [inline image_recognition_tensorflow/test/models/.gitignore]: would `models/*` in a `.gitignore` on parent level not be better?

File: image_recognition_tensorflow/test/test_retrain_and_evaluate.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+models_path = os.path.join(pkg_path, 'test/models')
+assets_path = os.path.join(pkg_path, 'test/assets')
+
+
+def test_retrain_and_evaluate():
+    assert 0 == os.system("rosrun {} retrain {} {} {}".format(pkg_name,
+                                                              os.path.join(assets_path, 'training'),
+                                                              models_path,
+                                                              models_path))
+    assert 0 == os.system("rosrun {} evaluate_classifier {} {} {}".format(pkg_name,
```
Comment (LoyVanBeek) [inline image_recognition_tensorflow/test/test_retrain_and_evaluate.py]: Why not add these two commands to travis? That also fails if the exit code != 0.

**Comment by LoyVanBeek** [comment]
Pro tip: GitHub apparently has file filters now in PRs, so you can ignore the gazillion .jpgs in the diff :ok_hand:

File: image_recognition_tensorflow/test/models/.gitignore (hunk 1 above)
Comment (reinzor) [reply image_recognition_tensorflow/test/models/.gitignore]: I like that there is a models folder on checkout

**Comment by LoyVanBeek** [comment]
Test is actually running now, that's nice and new right? 

It [crashes](https://travis-ci.org/tue-robotics/image_recognition/jobs/478840490#L3381) on ```[Errno 30] Read-only file system: '/root/catkin_ws/src/image_recognition/image_recognition_tensorflow/test/models/inception-2015-12-05.tgz```

**Comment by reinzor** [comment]
We have a build @LoyVanBeek :+1:

**Comment by LoyVanBeek** [comment]
LOL, without melodic but who cares right :laughing:

**Pull request merged**